File: Java-Core/src/main/java/presto/error/NullReferenceError.java
package presto.error;

import presto.expression.IExpression;
import presto.grammar.CategorySymbol;
import presto.runtime.Context;


public class NullReferenceError extends ExecutionError {

	private static final long serialVersionUID = 1L;

	@Override
	public IExpression getExpression(Context context) {
		return context.getRegisteredValue(CategorySymbol.class, "NULL_REFERENCE");
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/error/NullReferenceError.js
var ExecutionError = require("./ExecutionError").ExecutionError;

function NullReferenceError() {
	ExecutionError.call(this);
	return this;
}

NullReferenceError.prototype = Object.create(ExecutionError.prototype);
NullReferenceError.prototype.constructor = NullReferenceError;

NullReferenceError.prototype.getExpression = function(context) {
	return context.getRegisteredValue("NULL_REFERENCE");
};

exports.NullReferenceError = NullReferenceError;
